css/styles.css

```diff
@@ -223,3 +142,84 @@ select {
   margin-left: 10px;
   padding-bottom: 20px;
 }
+
+@media only screen and (max-width: 400px) {
+  .search-section {
+    background-size: 500%;
+    background-position: 45% 100%;
+  }
+}
+
+@media only screen and (max-width: 400px) {
+  .inline {
+    margin-left: 5%;
+    margin-top: 5%;
+  }
+}
+
+@media only screen and (max-width: 400px) {
+  .column-third {
+    width: 100%;
+  }
+}
+
+@media only screen and (max-width: 400px) {
+  .blue-area {
+    background-color: #f4f8fc;
+    border-radius: 50px;
+    padding-left: 15%;
+  }
+}
+
+@media only screen and (max-width: 400px) {
+  .chose-state {
+    border: none;
+    width: 110px;
+    margin-left: 5px;
+    color: black;
+    font-size: 10px;
+    padding: 15px;
+    border-radius: 10px;
+    background: #f1f4f8 url("http://cdn1.iconfinder.com/data/icons/cc_mono_icon_set/blacks/16x16/br_down.png") no-repeat 80px;
+  }
+}
+
+@media only screen and (max-width: 400px) {
+  .heading {
+    margin: 0;
+    color: white;
+    padding-top: 50%;
+    padding-left: 5%;
+    padding-bottom: 5px;
+    font-size: 38px;
+    font-family:
+      Fenix,
+      serif;
+    font-weight: normal;
+  }
+}
+
+@media only screen and (max-width: 400px) {
+  .search-box {
+    background-color: black;
+    margin-left: 5px;
+    color: white;
+    font-size: 10px;
+    padding: 15px 30px;
+    border-radius: 10px;
+    border: none;
+  }
+}
+
+@media only screen and (max-width: 400px) {
+  .city-name {
+    color: black;
+    padding: 15px;
+    width: 130px;
+    font-size: 10px;
+    border-radius: 10px;
+    border: none;
+    padding-left: 40px;
+    background: #f1f4f8 url("../images/123.png") no-repeat 2px;
+  }
+}
```

Comment:
All of these media queries are for the exact same size. You should merge them into one media query.